metricbeat/module/kibana/stats/_meta/fields.yml

```diff
@@ -16,11 +16,11 @@
       type: keyword
       description: >
         Name of Kibana's internal index
-    - name: host.name
+    - name: service.address
       type: keyword
       description: >
         Kibana instance hostname
-    - name: transport_address
+    - name: host.address
```

Comment:
This could be confusing if someone monitors Kibana from a remote server. It's a general problem we have.

Reply:
So this should be left as `transport_address` here?

Reply:
I made this change based on the conversation here: https://github.com/elastic/beats/pull/10218#discussion_r249962252

But now it sounds like we shouldn't make this change?

@ruflin @webmat could you please clarify whether this fields needs to be changed or left as-is? Thanks.

Reply:
I have second thoughts on this as it could cause issues  confusion on the user side. @webmat WDYT?